commands.go

```diff
@@ -129,6 +129,8 @@ type Cmdable interface {
 
 	Command(ctx context.Context) *CommandsInfoCmd
 	CommandList(ctx context.Context, filter *FilterBy) *StringSliceCmd
+	CommandGetKeys(ctx context.Context, command string, args ...string) *StringSliceCmd
+	CommandGetKeysAndFlags(ctx context.Context, command string, args ...string) *KeyFlagsCmd
```

Comment:
```go
...(ctx context.Context, commands ...interface{})
```

Reply:
Each of these commands would take only one type of command and their corresponding argument, correct?
In that case, can't we use a string for the command and then the args followed by it?

Reply:
Should we maintain by calling it commandAndArgs? @monkey92t

Reply:
We don't need to distinguish between commands and args.

Reply:
I was just trying to decide on apt. nomenclature for this, just commands also works.